source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -78,7 +82,32 @@ Settings
 
        |
        | **Default**: ``""``
-       | **Accepted Values**: ``""`` or ``"default"`` or ``"updateLookup"``
+       | **Accepted Values**: ``""``, ``"updateLookup"``, ``"whenAvailable"``, or ``"required"``
+       
+   * - | **change.stream.full.document.before.change**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Configures the document pre-image your change stream returns on update
+         operations. The pre-image is not available in source records
```

Comment:
Question:

Should this be "for source records" instead of "in source records"? Because it seems like we are not in the source record when we receive the pre-image.